samples/websocket/chatApp/bundle/resources/pom.xml

```diff
@@ -0,0 +1,437 @@
+<!--
```

Comment:
Bundle should be independent from carbon kernel. The bundle is intended to be copied to the dropins folder manually by the user. (This should be specified in the README)